packages/features/bookings/Booker/components/hooks/useOverlayCalendar.ts

```diff
@@ -1,27 +1,22 @@
-import { useSession } from "next-auth/react";
-import { useSearchParams } from "next/navigation";
 import { useEffect, useState } from "react";
 import { shallow } from "zustand/shallow";
 
 import dayjs from "@calcom/dayjs";
 import { useTimePreferences } from "@calcom/features/bookings/lib";
-import { localStorage } from "@calcom/lib/webstorage";
-import { trpc } from "@calcom/trpc/react";
 
-import { useBookerStore } from "../../store";
 import { useOverlayCalendarStore } from "../OverlayCalendar/store";
+import type { UseCalendarsReturnType } from "./useCalendars";
 import { useLocalSet } from "./useLocalSet";
 
-export type useOverlayCalendarReturnType = ReturnType<typeof useOverlayCalendar>;
-
-export const useOverlayCalendar = () => {
-  const utils = trpc.useContext();
-  const { set, toggleValue, hasItem, clearSet } = useLocalSet<{
+export const useOverlayCalendar = ({
```

Comment:
Should this be in OverlayCalendar/hooks/useOverlayCalendar instead? I think they belong together.